open-api/src/testFixtures/java/org/apache/iceberg/rest/RCKUtils.java

```diff
@@ -76,15 +76,21 @@ static Map<String, String> environmentCatalogConfig() {
                 HashMap::new));
   }
 
-  static RESTCatalog initCatalogClient() {
-    Map<String, String> catalogProperties = Maps.newHashMap();
+  public static RESTCatalog initCatalogClient() {
```

Comment:
I think this may be based off the Spark Tests PR?  Hopefully we can rebase and avoid this using `ExtensionContext`

Reply:
yep, this just needs a rebase after #11093 goes in